users/urls.py

```diff
@@ -2,9 +2,6 @@
 from . import views
 
 urlpatterns = [
-    path('profile/', views.profile, name='users-profile'),
-    path('login/', views.login, name='users-login'),
-    path('logout/', views.logout, name='users-logout'),
+    path('', views.profile, name='users-profile'),
     path('edit-profile/', views.edit_profile, name='users-edit-profile')
```

Comment:
I think we want to move edit-profile/ to a general url like you did with login and profile

Reply:
Done!